test/rmgpy/molecule/groupTest.py

```diff
@@ -1625,3 +1625,21 @@ def test_get_element_count(self):
         expected = {"C": 1, "N": 1, "P": 1}
         result = group.get_element_count()
         assert expected == result
+    
+    def test_is_reaction_bond(self):
+        """
+        Test the is_reaction_bond() function
+        """
+        group = Group().from_adjacency_list("""
+                                            1 R!H u1 {2,R} {3,vdW}
+                                            2 R u0 {1,R}
+                                            3 H u0 {vdW,1} {4,S}
```

Comment:
Why `{vdW,1}` but not `{1,vdW}`?

```suggestion
                                            3 H u0 {vdW,1} {4,S}
```

Reply:
Fixed